README.md

```diff
@@ -1,3 +1,13 @@
 # WordPress Performance
 
 Monorepo for the [WordPress Performance Group](https://make.wordpress.org/core/tag/performance/), primarily for the overall performance plugin, which is a collection of standalone performance modules.
+
+## Useful commands
+
```

Comment:
Would it makes sense to mention we need to have Docker up and running before running the `wp-env` commands. And/or link to the handbook (https://developer.wordpress.org/block-editor/reference-guides/packages/packages-env/)

Reply:
+1 to that, for someone without Docker this `npm run wp-env start` and everything after fails.

Reply:
Great point, I added a note about both Node.js and Docker being required.